autoware.repos

```diff
@@ -104,10 +104,16 @@ repositories:
     type: git
     url: https://github.com/tier4/nebula.git
     version: main
+  # Fork of transport_drivers that enables reduction of copy operations
   sensor_component/external/transport_drivers:
     type: git
-    url: https://github.com/MapIV/transport_drivers.git
-    version: boost
+    url: https://github.com/mojomex/transport_drivers
+    version: mutable-buffer-in-udp-callback
```

Comment:
Similarly, I've forked https://github.com/autowarefoundation/transport_drivers/tree/mutable-buffer-in-udp-callback too.

We can remove the external folder and keep it directly in `sensor_component/transport_drivers` folder.

Reply:
Addressed in [b9746a3](https://github.com/autowarefoundation/autoware/pull/5008/commits/b9746a3f15e4d1092f53d3d474005d3ec622c170) :+1: